src/main/resources/application.yaml

```diff
@@ -0,0 +1,12 @@
+spring:
+  main:
+    allow-bean-definition-overriding: true
+  r2dbc:
+    url: r2dbc:postgresql://address-db:5432/notification
+  flyway:
+    url: jdbc:postgresql://address-db:5432/notification
```

Comment:
`notification`?
```suggestion
    url: r2dbc:postgresql://address-db:5432/address
  flyway:
    url: jdbc:postgresql://address-db:5432/address
```

Reply:
tf, I searched globally with IntelliJ for this term
why did it ignore these two